vendor/github.com/openshift/installer/pkg/aro/dnsmasq/dnsmasq.go

```diff
@@ -108,6 +108,22 @@ chmod 0744 /etc/resolv.conf.dnsmasq
 /bin/rm $TMPNETRESOLV
 /bin/rm $TMPSELFRESOLV
 {{ end }}
+
```

Comment:
As of #2932, this file has been duplicated directly into the ARO-RP codebase rather than referenced from the vendored dependency.

If I'm understanding this correctly, these changes will need to be made to both:
- ARO-RP: https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/dnsmasq/dnsmasq.go
- installer-aro: https://github.com/jewzaam/installer-aro/blob/release-4.11-azure/pkg/aro/dnsmasq/dnsmasq.go

as this specific vendored file is no longer referenced by our operator reconciler.

Reply:
@tsatam Thank you for bringing this to my attention!